Mathlib/Data/Seq/WSeq.lean

```diff
@@ -1158,7 +1158,6 @@ theorem liftRel_flatten {R : α → β → Prop} {c1 : Computation (WSeq α)} {c
   ⟨S, ⟨c1, c2, rfl, rfl, h⟩, fun {s t} h =>
     match s, t, h with
     | _, _, ⟨c1, c2, rfl, rfl, h⟩ => by
-      -- Porting note: `exists_and_left` should be excluded.
```

Comment:
I don't understand what that porting note meant or why we can safely remove it now. Can you explain?

Reply:
In mathlib3, it was
```lean
    simp, apply lift_rel_bind _ _ h,
```
which was ported as
```
      -- Porting note: `exists_and_left` should be excluded.
      simp [-exists_and_left]; apply liftRel_bind _ _ h
```
until #11259 squeezed the simp